### PR TITLE
Readme update

### LINKED ISSUE #414: improvements of the graph in the README
- it's not super clear that the EthereumPermitBundler only inherits from PermitBundler
- there is a typo Etherem => Ethereum

### PR DESCRIPTION
**Pull request by MathisGD**
Fixes #414, #436

### CODE REVIEW

**Review by MerlinEgalite — APPROVED** [review]

**Review by adhusson — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -4,14 +4,19 @@
...
 
 Each Bundler is a domain-specific abstract layer of contract that implements some functions that can be bundled in a single call by EOAs to a single contract. They all inherit from [`CoreBundler`](./src/CoreBundler.sol) that enables bundling multiple function calls into a single `multicall(bytes[] calldata data)` call to the end bundler contract. Each chain-specific bundler is available under their chain-specific folder (e.g. [`ethereum`](./src/ethereum/)).
 
 Some chain-specific domains are also scoped to the chain-specific folder, because they are not expected to be used on any other chain (e.g. DAI and its specific `permit` function is only available on Ethereum - see [`EthereumPermitBundler`](./src/ethereum/EthereumPermitBundler.sol)).
 
 User-end bundlers are provided in each chain-specific folder, instanciating all the intermediary domain-specific bundlers and associated parameters (such as chain-specific protocol addresses, e.g. [`EthereumBundlerV2`](./src/ethereum/EthereumBundlerV2.sol)).
 
+## Deployments
+
+- [EthereumBundlerV2](https://github.com/morpho-org/morpho-blue-bundlers/releases/tag/v1.2.0)
```
Comment (QGarchery) [inline README.md:17]: Should we link to V1 also ? I did not find any deployment address related to it on the releases

**Review by adhusson — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -14,8 +14,9 @@ User-end bundlers are provided in each chain-specific folder, instanciating all
 
 ## Deployments
 
+- [EthereumBundlerV2](https://github.com/morpho-org/morpho-blue-bundlers/releases/tag/v1.0.0)
```
Comment (QGarchery) [inline README.md:17]:
```suggestion
- [EthereumBundler](https://github.com/morpho-org/morpho-blue-bundlers/releases/tag/v1.0.0)
```
or 
```suggestion
- [EthereumBundlerV1](https://github.com/morpho-org/morpho-blue-bundlers/releases/tag/v1.0.0)
```
but I think it's better to put the name of the contract directly which is `EthereumBundler`

File: README.md [hunk 3]
```diff
@@ -4,14 +4,20 @@
 
 ## Structure
 
-![image (4)](https://github.com/morpho-org/morpho-blue-bundlers/assets/44097430/5cb0796b-c20c-415e-840d-8b0705836dc8)
+![bundler-2 drawio](https://github.com/morpho-org/morpho-blue-bundlers/assets/74971347/59549998-45e5-4154-a93e-6d684247105b)
```
Comment (QGarchery) [inline README.md:7]: Note that it will have to be adapted after #441 because of the revert of the renaming of Core Bundler

**Review by QGarchery — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -4,14 +4,20 @@
 
 ## Structure
 
-![image (4)](https://github.com/morpho-org/morpho-blue-bundlers/assets/44097430/5cb0796b-c20c-415e-840d-8b0705836dc8)
+![bundler](https://github.com/morpho-org/morpho-blue-bundlers/assets/74971347/965feec8-cd15-488e-b97f-644caef96bfc)
```
Comment (adhusson) [inline README.md:7]: Inheritance graph <> code discrepancy: 
* `BaseBundlerV2` does not inherit `PermitBundler` 
* `EthereumBundlerV2` does not inherit `PermitBundler`

Also, the arrows coming out of `StEthBundler` and `WNativeBundler` have almost exactly the same color.

**Review by adhusson — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (MathisGD) [reply README.md:7]: `ChainAgnosticBundlerV2` [inherits](https://github.com/morpho-org/morpho-blue-bundlers/blob/7be911d9148f280b093074381000a6d7d566ff7e/src/chain-agnostic/ChainAgnosticBundlerV2.sol#L20), and on my graph EthereumBundlerV2 does not inherit from PermitBundler. Anyway I made the arrows bigger, to make it clearer

**Review by adhusson — APPROVED** [review]

File: README.md
Comment (QGarchery) [inline README.md:1]: Given the missed dependencies (and renaming) in the graph, this makes me think that a solution where they appear in a text form would be more tractable. Something like [this](https://github.com/morpho-org/metamorpho/tree/main/certora#specification-imports-graph) for example. 
This is just a remark though, potentially to do in another PR or even on other repos

**Review by QGarchery — APPROVED** [review]

File: README.md
Comment (MathisGD) [reply README.md:1]: I tried it (#424) and it's almost unreadable. Maybe with a bit more creativity (like grouping some contracts) it could have been.

**Pull request merged**